fbw-a380x/src/wasm/systems/a380_systems/src/air_conditioning/mod.rs

```diff
@@ -398,17 +428,27 @@ impl A380AirConditioningSystem {
             )
         });
 
-        for fan in self.cabin_fans.iter_mut() {
-            fan.update(cabin_simulation, &self.acsc.cabin_fans_controller())
-        }
+        self.tadd.update(
+            context,
+            &self.air_conditioning_overhead,
+            cpiom_b,
+            &self.trim_air_system,
+            engines,
+            pneumatic,
+            pneumatic_overhead,
+            cpiom_b.should_close_taprv(),
+        );
+
+        self.vcm.iter_mut().for_each(|module| {
+            module.update(&self.air_conditioning_overhead, pressurization_overhead)
+        });
+
+        self.fan_update(cabin_simulation, cpiom_b);
+
+        let pack_flow: [MassRate; 2] = [self.fdac[0].pack_flow(), self.fdac[1].pack_flow()];
 
-        let pack_flow: [MassRate; 2] = [
-            self.acsc.individual_pack_flow(Pack(1)),
-            self.acsc.individual_pack_flow(Pack(2)),
-        ];
-        let duct_demand_temperature = self.acsc.duct_demand_temperature();
         for (id, pack) in self.packs.iter_mut().enumerate() {
-            pack.update(pack_flow[id], &duct_demand_temperature)
+            pack.update(pack_flow[id], &cpiom_b.duct_demand_temperature())
         }
```

Comment:
```suggestion
        for (pack, pack_flow) in self
            .packs
            .iter_mut()
            .zip(self.fdac.iter().map(|fdac| fdac.pack_flow()))
        {
            pack.update(pack_flow, &cpiom_b.duct_demand_temperature())
        }
```
Unfortunately this suggestion is a bit weirdly formatted because it also includes the deleted lines.

Reply:
I like it! Thanks.